gov2/kms/ReEncryptData/README.md

```diff
@@ -0,0 +1,10 @@
+### ReEncryptData/ReEncryptDatav2.go
+
+This example reencrypts some text using an AWS Key Management Service (AWS KMS) customer master key (CMK).
+
+`go run ReEncryptDatav2.go -k KeyID -d DATA`
+
+- _KeyID_ is the ID of the AWS KMS key to use for reencrypting the data.
+- _DATA_ is the data, to reencrypt, as a string.
```

Comment:
_DATA_ is the data to reencrypt
(remove the ',' between 'data' and 'to')

Reply:
Fixed